internalsdk/session_model.go

```diff
@@ -185,11 +191,34 @@ func NewSessionModel(mdb minisql.DB, opts *SessionModelOpts) (*SessionModel, err
 	m.baseModel.doInvokeMethod = m.doInvokeMethod
 	if opts.Platform == "ios" {
 		m.SetAuthEnabled(true)
+		go m.setupIosConfigure(opts.ConfigPath, int(userID), token, deviceID)
 	}
 	go m.initSessionModel(context.Background(), opts)
 	return m, nil
 }
 
+// setupIosConfigure sets up the iOS configuration for the session model.
+// It continuously checks if the global configuration is available and retries every second if not.
+func (m *SessionModel) setupIosConfigure(configPath string, userId int, token string, deviceId string) {
+	go func() {
+		cf := ios.NewConfigurer(configPath, userId, token, deviceId, "")
+		for {
+			if cf.HasGlobalConfig() {
+				global, _, _, err := cf.OpenGlobal()
+				if err != nil {
+					log.Errorf("Error while opening global %v", err)
+					return
+				}
+				m.iosConfigurer = global
+				log.Debugf("Found global config IOS configure done %v", global)
+				break
+			}
+			time.Sleep(1 * time.Second)
```

Comment:
Are we able to use a ticker.Timer in a select instead here to periodically check the configuration to avoid busy waiting?

Reply:
Done.